package.json

```diff
@@ -42,10 +43,18 @@
   },
   "devDependencies": {
     "@angular-devkit/build-angular": "~13.1.2",
+    "@angular-eslint/builder": "13.1.0",
+    "@angular-eslint/eslint-plugin": "13.1.0",
+    "@angular-eslint/eslint-plugin-template": "13.1.0",
+    "@angular-eslint/schematics": "13.1.0",
+    "@angular-eslint/template-parser": "13.1.0",
     "@angular/cli": "~13.1.2",
     "@angular/compiler-cli": "~13.1.1",
     "@types/jasmine": "~3.10.3",
     "@types/node": "^17.0.8",
+    "@typescript-eslint/eslint-plugin": "5.11.0",
+    "@typescript-eslint/parser": "5.11.0",
+    "eslint": "^7.1.0",
```

Comment:
Is there a reason not to use latest version of packages ?

Reply:
it was because I had an error creating the ESLint constructor but obviously I just retested and it now works with the latest version...
I upgraded

Reply:
However, I still upgraded to version 7.32 which seems to work fine.